src/crypto/chain.js

```diff
@@ -39,6 +45,54 @@ export class AddressGenerator {
   }
 }
 
+// TODO: make ShelleyAddressGenerator a subclass of AddressGenerator
+export class ShelleyAddressGenerator {
+  addressChain: Bip32PublicKey
+  // stakingKey: PublicKey
+  // for now, need to store accountKey instead of stakingKey because the latter
+  // is PublicKey instance and from_bech32() is currently mising from the
```

Comment:
is this still missing?

Reply:
Yes, but finally I will just store the account-level public key, as we did in Byron.